ceresdb-example/src/test/java/io/ceresdb/ReadmeTest.java

```diff
@@ -51,51 +50,106 @@ public void readmeTest() throws ExecutionException, InterruptedException {
-                                + "c2 STRING TAG NOT NULL," + "c3 DOUBLE NULL," + "c4 STRING NULL," + "c5 INT64 NULL,"
-                                + "c6 FLOAT NULL," + "c7 INT32 NULL," + "c8 INT16 NULL," + "c9 INT8 NULL,"
-                                + "c10 BOOLEAN NULL," + "c11 UINT64 NULL," + "c12 UINT32 NULL," + "c13 UINT16 NULL,"
-                                + "c14 UINT8 NULL," + "c15 TIMESTAMP NULL," + "c16 VARBINARY NULL,"
-                                + "TIMESTAMP KEY(ts)" + ") ENGINE=Analytic";
-        SqlResult result = client.management().executeSql(createTableSql);
+        String createTableSql = "CREATE TABLE IF NOT EXISTS machine_table(" +                                                                                              "ts TIMESTAMP NOT NULL," + //
+                "ts TIMESTAMP NOT NULL," +
+                "city STRING TAG NOT NULL," +
+                "ip STRING TAG NOT NULL," +
+                "cpu DOUBLE NULL," +
+                "mem DOUBLE NULL," +
+                "TIMESTAMP KEY(ts)" + // timestamp column must be specified
+                ") ENGINE=Analytic";
+
+        Result<SqlQueryOk, Err> createResult = client.sqlQuery(new SqlQueryRequest(createTableSql)).get();
+        if (!createResult.isOk()) {
+            throw new IllegalStateException("Fail to create table");
+        }
 
         final long t0 = System.currentTimeMillis();
         final long t1 = t0 + 1000;
-        final long t2 = t1 + 1000;
-        final Rows data = Series.newBuilder("machine_metric").tag("city", "Singapore").tag("ip", "127.0.0.1")
-                .toRowsBuilder()
-                // codes below organizes 3 lines data (3 timestamps) for the `cpu` and `mem` column, this will just transport once through network. CeresDB encourage practices like this, because the SDK could use efficient compression algorithm to reduce network traffic and also be friendly to the sever side.
-                .field(t0, "cpu", FieldValue.withDouble(0.23)) // first row, first column
-                .field(t0, "mem", FieldValue.withDouble(0.55)) // first row, second column
-                .field(t1, "cpu", FieldValue.withDouble(0.25)) // second row, first column
-                .field(t1, "mem", FieldValue.withDouble(0.56)) // second row, second column
-                .field(t2, "cpu", FieldValue.withDouble(0.21)) // third row, first column
-                .field(t2, "mem", FieldValue.withDouble(0.52)) // third row, second column
+        final List<Point> data = Point.newPointsBuilder("machine_table")
+                .addPoint() // first point
+                    .setTimestamp(t0)
+                    .addTag("city", "Singapore")
+                    .addTag("ip", "10.0.0.1")
+                    .addField("cpu", Value.withDouble(0.23))
+                    .addField("mem", Value.withDouble(0.55))
+                    .build()
```

Comment:
It is a little strange that`addPoint` method returns `PointBuilder' instead of add one read point, and `build()` of `PointBuilder' returns `PointsBuilder`.

Reply:
remve `PointsBuilder`, and Add `PointBuilder` for single point now.